spec/controllers/business_entities_controller_spec.rb

```diff
@@ -130,4 +130,35 @@
       it { is_expected.to render_template(:edit) }
     end
   end
+
+  describe 'GET #confirm' do
+    let(:business_entity) { office.business_entities.first }
+    subject { response }
+    before do
+      sign_in admin
+      get :confirm, office_id: office.id, id: business_entity.id
+    end
+
+    it { is_expected.to have_http_status(:success) }
+
+    it { is_expected.to render_template(:confirm) }
+
+    it 'assigns the @business_entity variable' do
+      expect(assigns(:business_entity)).to be_a_kind_of BusinessEntity
```

Comment:
Here I think you can test very specifically that `expect(assigns(:business_entity)).to eql(business_entity)`.